apps/admin-be/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "admin-be",
+  "version": "1.0.0",
+  "description": "backend for admin panel",
+  "main": "dist/index.js",
+  "license": "MIT",
+  "scripts": {
+    "devStart": "nodemon --ignore tests/ --watch src -e ts,tsx --exec ts-node src/index.ts",
+    "build-ts": "tsc",
+    "lint": "tsc --noEmit && eslint \"**/*.{js,ts}\" --quiet --fix",
+    "serve": "node dist/index.js",
+    "build": "yarn run lint && yarn run build-ts",
```

Comment:
can remove the lint job here, let's call tsc directly. This makes the `build-ts` script redundant.